### ISSUE TITLE
wrong height of entire grid efter resize

### ISSUE THREAD

**Issue by badtant** [ducksboard/gridster.js#179]
When I resize a widget to a large number of rows "set_dom_grid_height" sets the wrong height of the grid.

The bug I'm having right now is that i only have one widget in my grid and I'm resizing it to height 37 but "get_highest_occupied_cell" returns 15 to "set_dom_grid_height" which causes my widget to overflow the grid.

Any ideas?

**Comment by badtant** [comment]
does it have anyting with min_rows to do? can't see why though...

**Comment by Abhoryo** [comment]
Yeah, that's can be because of the min_rows value too low or maybe widget_margins and widget_base_dimensions haven't compatible values with the container of your gridster.

Ex: You can't set max_cols = 2, column width = 305px and widget margin width = 5px if your container is lower than 630px
